_redirects

```diff
@@ -0,0 +1 @@
+/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms/ /docs/fanatical-support-aws/user-management-and-perms/
```

Comment:
Redirect not yet working: https://deploy-preview-111--docs-aws.netlify.app/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms/

with try removing trailing slash, could be that the file is not getting picked up by netlify

```suggestion
/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms /docs/fanatical-support-aws/user-management-and-perms
```